api.md

```diff
@@ -0,0 +1,224 @@
+# API Documentation
+
+The poker server runs a poker game with which the client interacts. The
+interaction is done over HTTP for ease of use, but it is worth noting that the
+entirety of this interaction is stateful and that very little (if anything)
+can be meaningfully cached.
+
+## Transport and Session
+
+The data sent and received by the server will be with JSON. Attached to many
```

Comment:
> Let's simply use a "Deck ID" associated with which Deck the client is connected to.

@semmypurewal isn't that functionally the same? I suppose `/restart-game` would have to be different in that case, and it might be better to nix the endpoint altogether.